File: Factorial.cs
using System;

class MainClass {
  public static int FirstFactorial(int num) { 
  
    // code goes here  
    /* Note: In C# the return type of a function and the 
       parameter types being passed are defined, so this return 
       call must match the return type of the function.
       You are free to modify the return type. */
    if (num == 1)
        return 1;
    return num * FirstFactorial(--num);
            
  }

  static void Main() {  
    // keep this function call here
    Console.WriteLine(FirstFactorial(Console.ReadLine()));
  } 
   
}

File: Factorial.cpp
#include <iostream>
using namespace std;

int FirstFactorial(int num) { 

  // code goes here
  int out = 1;
  for (int i=num; i > 0; i--) {
  	out *= i;
  }
  return out; 
            
}

int main() { 
  
  // keep this function call here
  cout << FirstFactorial(gets(stdin));
  return 0;
    
}